camel-component/pom.xml

```diff
@@ -0,0 +1,165 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.debezium.examples.camel</groupId>
+    <artifactId>component-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>Parent for Q&amp;A Camel Debezium example</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+
+        <database.hostname>localhost</database.hostname>
+        <database.port>3306</database.port>
```

Comment:
Perhaps confusing to use the MySQL port?

Reply:
@gunnarmorling It is hardoded now, thanks for poinitng it out.

Reply:
Ok, can this be removed then?